tests/test_app/TestApp/Model/Table/ArticlesTable.php

```diff
@@ -19,7 +19,7 @@
  */
 class ArticlesTable extends Table
 {
-    public function initialize(array $config)
+    public function initialize(array $config): void
```

Comment:
I think methods that are meant to always be overridden by the user should not have these type of changes.

Reply:
Isnt it especially important here to not expect it to return and follow right signature?

Reply:
I’m not sure that it is specially important to be honest. It is nice to have, but this particular method does not create a lot of confusion about what the return value should be

Reply:
So why not having it?
The IDE will more easily be able to completely typehint it if the parent method already is fully equipped here.

Reply:
Most importantaly code becomes consistent. Doing it sometimes, sometimes not is confusing to everyone IMO. And sniffers in the future can't be used to assert and auto-fix.